packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -62,26 +63,25 @@ function overrideEnv<T extends execa.Options>(options?: T): T | undefined {
  * When using this methid, the subprocess should be started with buffer: false option
  * as there's no need for allocating memory for the output that's going to be very long.
  */
-export function lineReader(childProcess: ExecaChildProcess<string>, includeStderr = false) {
-  const input = childProcess.stdout;
-  if (!input) {
-    throw new Error("Child process has no stdout");
+export function lineReader(childProcess: ExecaChildProcess<string>) {
+  if (!childProcess.stdout) {
+    throw new Error("Child process doesn't have stdout");
```

Comment:
is it possible for the process to not have stdout but have stderr? If so and if we decide that stderr should be passed on by default, maybe it would be useful to allow line reader to exist without stdout?

Reply:
This is only possible if the process is configured to pipe stdout. We typically use lineReader to read from stdout hence it crashes if stdout is not present. If we ever need to reac stderr and not stdout we may need to change this function to support such usecase.